docs/main-entrypoint.md

```diff
@@ -15,5 +15,44 @@ module.exports = async function service(fastify) {
   })
 }
 ```
+As you can see the function must be declared `async` and must be exported as the root of the module.
 
-As you can see the function must be declared `async` and must be exported as the root of the module.  
+## Custom Status Routes
+With `lc39` your service will automatically inherit **two** fixed routes for getting infomation on the
+service:
+
+- `GET /-/healthz`
```

Comment:
`healthy`? just because then you get `ready` and `healthy`, looks more consistent.

Reply:
For me the names can be further discussed, maybe on the next coffee break 😄